sycl/include/sycl/ext/intel/esimd/common.hpp

```diff
@@ -545,7 +545,8 @@ template <cache_hint Hint> class cache_hint_wrap {
   }
 };
 
-constexpr bool are_both(cache_hint First, cache_hint Second, cache_hint Val) {
+template <cache_hint Val>
+constexpr bool are_all(cache_hint First, cache_hint Second) {
```

Comment:
if we change this from `are_both` to `are_all`, i would expect it to take in variadic arguments

Reply:
Nick, thank you, It seems not critical now, right, taking into account the approval? Let me do that extra-fix in one of follow-up commits.